mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1441,9 +1439,9 @@ def Rock_ThreadwiseGemmOp
   let hasVerifier = 1;
 }
 
-// threadwise_accel_gemm
-def Rock_ThreadwiseAccelGemmOp
-    : Rock_Op<"threadwise_accel_gemm",
+// threadwise_gemm_accel
+def Rock_ThreadwiseGemmAccelOp
+    : Rock_Op<"threadwise_gemm_accel",
```

Comment:
Suggestion for future:
Try to keep independent changes in seperate PRs for faster reviews.